### PR TITLE
Cant upload on SSO

### PR DESCRIPTION
**Pull request by AStoerkel1**
:fire::bug::fire:

### CODE REVIEW

File: src/app/clark.component.ts [hunk 1]
```diff
@@ -83,7 +84,8 @@ export class ClarkComponent implements OnInit {
     private toaster: ToastrOvenService,
     private view: ViewContainerRef,
     private cookieAgreement: CookieAgreementService,
-    private subscriptionAgreement: SubscriptionAgreementService
+    private subscriptionAgreement: SubscriptionAgreementService,
+    private cookies: CookieService
```
Comment (mfranz1) [inline src/app/clark.component.ts:88]: Not used

File: src/app/clark.component.ts [hunk 2]
```diff
@@ -14,6 +14,7 @@ import { ToastrOvenService } from './shared/modules/toaster/notification.service
 import { CookieAgreementService } from './core/cookie-agreement.service';
 import { SubscriptionAgreementService } from './core/subscription-agreement.service';
 import { NavbarService } from './core/navbar.service';
+import { CookieService } from 'ngx-cookie-service';
```
Comment (mfranz1) [inline src/app/clark.component.ts:17]: Not used

**Review by mfranz1 — CHANGES_REQUESTED** [review]

**Review by EVogel1999 — COMMENTED** [review]
Just mike's comments.  Looks good otherwise!

**Review by JosueMolinaMorales — APPROVED** [review]

**Review by mfranz1 — APPROVED** [review]

**Comment by mfranz1** [comment]
Can you throw a patch version up

**Pull request merged**